package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kompasid/web-components",
-  "version": "v1.5.11",
+  "version": "v1.5.12",
```

FYI kita udah di v1.5.13, jadi disarankan update ke v1.5.14
```suggestion
  "version": "v1.5.14",
```